infra/static/output.tf

```diff
@@ -3,3 +3,26 @@
 output "identity_zone_name_servers" {
   value = aws_route53_zone.identity.name_servers
 }
+
+# SES
+
+output "wellcomecollection_org_ses_vertification_token" {
+  value = {
+    name = "_amazonses.${aws_ses_domain_identity.wellcomecollection_org.id}"
+    type = "TXT"
+    records = [
```

Comment:
Needs a `terraform fmt` I think.

Reply:
`terraform fmt` is fine with it actually, or it wouldn't have passed the build! Fixed it anyway.